modules/programs/zsh/zsh-abbr.nix

```
@@ -0,0 +1,40 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.programs.zsh.zsh-abbr;
+in {
+  meta.maintainers = [ hm.maintainers.ilaumjd ];
+
+  options.programs.zsh.zsh-abbr = {
+    enable =
+      mkEnableOption "zsh-abbr - zsh manager for auto-expanding abbreviations";
+
+    abbrs = mkOption {
```

Maybe can expand the name for clarity?

```suggestion
    abbreviations = mkOption {
```